domains/ris.json

```
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "ris",
```

```suggestion
    "username": "yowRiss",
```
This must be your GitHub username.